bedrock/settings/base.py

```diff
@@ -584,7 +584,7 @@ def lazy_email_backend():
 }
 
 # Google Analytics
-GA_ACCOUNT_CODE = ''
+GA_ACCOUNT_CODE = 'UA-36116321-20'
```

Comment:
Do we want to put this account code here? Should probably go in `.env` locally, right?